src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
```

Comment:
```suggestion
export async function getComments(postId: Post['id']) {
```
Треба всюди пробігтись і переписати такі місця на вже існуючі типи